### PR TITLE
frontend: UI fix & estimated rewards

### PR DESCRIPTION
**Pull request by leonkho**
- Added estimated rewards on staking pool card
- Fixed border-radius issue on portfolio position main container

### CODE REVIEW

File: webapp/src/components/Staking/StakingPools.tsx [hunk 1]
```diff
@@ -370,8 +387,18 @@ const StakingPool: React.FC<StakingPoolProps> = ({ vaultOption }) => {
...
           </div>
 
-          {/* Pool reward of duration */}
+          {/* Estimated pool rewards */}
           <div className="d-flex align-items-center mt-4 w-100">
+            <div className="d-flex align-items-center">
+              <SecondaryText>Your estimated rewards </SecondaryText>
+            </div>
+            <PoolRewardData className="ml-auto" color={color}>
+              {renderEstimatedRewards()} RBN
```
Comment (kenchangh) [inline webapp/src/components/Staking/StakingPools.tsx:396]: Can we make the RBN number 2 decimal places?

Comment (kenchangh) [inline webapp/src/components/Staking/StakingPools.tsx:396]: Can you round up the number to nearest integer so we can remove the decimal place entirely?

Comment (leonkho) [reply webapp/src/components/Staking/StakingPools.tsx:395]: Done!

**Review by kenchangh — APPROVED** [review]
LGTM

**Pull request merged**